SwiftLeeds/Views/Local/BottomSheetView.swift

```diff
@@ -73,14 +79,26 @@ struct BottomSheetView: View {
     }
 }
 
+private extension BottomSheetView {
+    struct Constant {
+        static let tabBarPadding: CGFloat = UITabBar().frame.height
```

Comment:
This makes me cringe a bit but I've never found a way around getting the exact number in SwiftUI 🤔